.travis.yml

```diff
@@ -41,32 +41,21 @@ matrix:
       env: PROJECT_DIR=examples/MySQL-client TOOLCHAIN=clang-libstdcxx
     - os: linux
       env: PROJECT_DIR=examples/MySQL-client TOOLCHAIN=gcc-4-8
-
-    #  FIXME:
-    #  * https://travis-ci.org/ingenue/hunter/jobs/112505605
-    # - os: linux
-    #   env: PROJECT_DIR=examples/MySQL-client TOOLCHAIN=android-ndk-r10e-api-19-armeabi-v7a-neon
-
-    #  FIXME:
-    #  * https://travis-ci.org/ingenue/hunter/jobs/112505606
-    # - os: linux
-    #   env: PROJECT_DIR=examples/MySQL-client TOOLCHAIN=analyze
-
-    #  FIXME:
-    #  * https://travis-ci.org/ingenue/hunter/jobs/112505607
-    # - os: linux
-    #   env: PROJECT_DIR=examples/MySQL-client TOOLCHAIN=sanitize-address
-
-    #  FIXME:
-    #  * https://travis-ci.org/ingenue/hunter/jobs/112505608
-    # - os: linux
-    #   env: PROJECT_DIR=examples/MySQL-client TOOLCHAIN=sanitize-leak
-
-    #  FIXME:
-    #  * https://travis-ci.org/ingenue/hunter/jobs/112505609
-    # - os: linux
-    #   env: PROJECT_DIR=examples/MySQL-client TOOLCHAIN=sanitize-thread
-
+    # FIXME: https://travis-ci.org/caseymcc/hunter/jobs/218630987
```

Comment:
probably didn't need to update this, original I thought I had gotten the osx-10-9 toolchain working so I made changes to the file with the intention to updating.